.circleci/config.yml

```diff
@@ -28,7 +28,25 @@ jobs:
       - run:
           name: "npm-pack"
           command: npm pack
-      # persist current folder for checkout and artifacts
+      - run:
+          name: "set-npm-version" 
+          command: |
+            echo "export NPM_PACKAGE_VERSION=$(npm pkg get version)" >> $BASH_ENV
+      - run: 
+          name: "set-is-beta"
+          command: |
+            if [[ $NPM_PACKAGE_VERSION == *"beta"* ]];
+            then
+              echo "export IS_BETA=true" >> $BASH_ENV
+            else
+              echo "export IS_BETA=false" >> $BASH_ENV
+            fi
+      - run:
+          name: "export-env"
+          command: |
+            echo "Will publish version: $NPM_PACKAGE_VERSION, is beta: $IS_BETA"
+            cp $BASH_ENV bash.env
```

Comment:
pass env vars down to next job (deploy-npm) following this guide:
https://support.circleci.com/hc/en-us/articles/10816400480411-How-to-pass-environment-variables-between-jobs